server/src/main/java/org/opensearch/index/translog/Translog.java

```diff
@@ -1817,6 +1817,13 @@ protected void setMinSeqNoToKeep(long seqNo) {}
 
     protected void onDelete() {}
 
+    /**
+     * Drains ongoing syncs to the underlying store. It returns a releasable which can be closed to resume the syncs back.
+     */
+    protected Releasable drainSync() {
+        return () -> {}; // noop
+    }
+
```

Comment:
Will prefer this abstract and make LocalTranslog noop instead so that new implementors explicitly override this